test/MadeTech/CacheWarming/Test/Unit/Config/ArrayConfigProviderTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace MadeTech\CacheWarming\Test\Unit\Config;
+
+
+use MadeTech\CacheWarming\Config\ArrayConfigProvider;
+use MadeTech\CacheWarming\Config\HrefLangExpansion;
+
+class ArrayConfigProviderTest extends \PHPUnit_Framework_TestCase
+{
+
+    /** @test * */
+    public function testThatItReturnsTheConfigPassedIn()
+    {
+        $configProvider = new ArrayConfigProvider(['a', 'b', 'c']);
+        $this->assertEquals(['a', 'b', 'c'], $configProvider->getSiteMapUrls());
+    }
+
+    /** @test * */
+    public function givenHrefLangExpansionInConfig_ThenDoNotProvideForCallToGetSiteMapUrls()
+    {
+        $configProvider = new ArrayConfigProvider([new HrefLangExpansion('', '', '')]);
+        $this->assertEquals([], $configProvider->getSiteMapUrls());
+    }
+
+    /** @test * */
+    public function givenNoHrefLangExpansionInConfig_ThenShouldReturnEmptyArray()
+    {
+        $configProvider = new ArrayConfigProvider(['a', 'b', 'c']);
+        $this->assertEquals([], $configProvider->getHrefLangExpansions());
+    }
+
+    /** @test * */
+    public function givenHrefLangExpansionInConfig_ThenShouldReturnIt()
+    {
+        $expansion = new HrefLangExpansion('', '', '');
```

Comment:
See the usage in the acceptance test, which better shows the behaviour coming together.

https://github.com/madetech/cache-warming/pull/9/files#diff-58ab4fef47798389b8972756f2e60b79R137